pkg/antctl/command_definition_test.go

```diff
@@ -45,6 +45,11 @@ type Foobar struct {
 	Foo string `json:"foo"`
 }
 
+var (
+	AntreaPolicyTierPriority = int32(250)
+	AntreaPolicyPriority     = float64(1.0)
+)
```

Comment:
these could be `const`

but I don't see a good reason to define them at all actually, since you are hardcoding the values in the expected output string anyway...

Reply:
Need to take addresses of these